package.json

```diff
@@ -1,48 +1,84 @@
-    "@rsksmart/rlogin": "^1.4.0-beta.1",
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
-    "@types/node": "^17.0.19",
-    "@types/react": "^17.0.39",
-    "@types/react-dom": "^17.0.11",
-    "abi-decoder": "^2.4.0",
-    "materialize-css": "^1.0.0-rc.2",
-    "patch-package": "~6.4.6",
-    "qrcode.react": "^3.0.2",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-materialize": "^3.10.0",
-    "react-scripts": "4.0.3",
-    "typescript": "^4.5.5",
-    "web-vitals": "^1.0.1",
-    "web3": "1.2.6",
-    "web3-utils": "1.2.6"
+  "description": "This project contains a sample dApp to interact with the RIF Relay project.",
+  "keywords": [
+    "relay",
+    "services"
+  ],
+  "bugs": {
+    "url": "https://github.com/rsksmart/rif-relay-sample-dapp/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rsksmart/rif-relay-sample-dapp"
   },
+  "license": "ISC",
+  "author": "RIF Payments Team",
+  "contributors": [
+    {
+      "name": "Raúl Laprida",
+      "url": "https://github.com/raullaprida"
+    },
+    {
+      "name": "Marcelo Ortelli",
+      "url": "https://github.com/mortelli"
+    },
+    {
+      "name": "Julian Len",
+      "url": "https://github.com/julianlen"
+    },
+    {
+      "name": "Wilson Hamilton",
+      "url": "https://github.com/wilsoniovlabs"
+    },
+    {
+      "name": "Antonio Morrone",
+      "url": "https://github.com/antomor"
+    },
+    {
+      "name": "Jose Jimenez",
+      "url": "https://github.com/jd-iov"
+    },
+    {
+      "name": "David Ugbero",
+      "url": "https://github.com/david-iov"
+    },
+    {
+      "name": "Juraj Piar",
+      "url": "https://github.com/jurajpiar"
+    },
+    {
+      "name": "Andrés Quijano",
+      "url": "https://github.com/AndresQuijano"
+    },
+    {
+      "name": "Francisco Tobar",
+      "url": "https://github.com/franciscotobar"
+    },
+    {
+      "name": "Christos Otarola",
+      "url": "https://github/ironFe93"
+    }
+  ],
   "scripts": {
-    "start": "react-scripts start",
     "build": "GENERATE_SOURCEMAP=false react-scripts build",
     "build:qa": "bash scripts/cli-process-env.sh 'qa' && npm run build",
     "build:testnet": "bash scripts/cli-process-env.sh 'testnet' && npm run build",
-    "test": "react-scripts test",
     "eject": "react-scripts eject",
+    "format": "npx prettier --check . !.gitignore  --check --ignore-unknown",
+    "format:fix": "npm run format -- --write --ignore-unknown",
     "lint": "eslint --ext .ts --ext .tsx .",
-    "lint:fix": "eslint . --ext .ts --ext .tsx --fix",
-    "prettier": "npx prettier --check .",
-    "prettier:fix": "npx prettier --write --ignore-unknown .",
-    "prepare": "patch-package && husky install"
+    "lint:fix": "npm run lint -- --fix",
+    "prepare": "scripts/prepare",
+    "start": "GENERATE_SOURCEMAP=false react-scripts start",
```

Comment:
The GENERATE_SOURCEMAP is set to false since since on react18 its showing multiple warnings, we need to identify how to resolve those warnings.